visual-js/visual-nightwatch/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:18 AS runner
+
+WORKDIR app
+
+COPY tsconfig.prod.json .
+COPY tsconfig.json .
+COPY package.json .
+
+COPY ./visual-nightwatch/src ./visual-nightwatch/src
+COPY ./visual-nightwatch/package.json ./visual-nightwatch/package.json
```

Comment:
IMO I'd like to see the same recommendation from the other PR here too -- simplify these files a bit.

Reply:
done. thanks for the suggestion